<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameters</class>
 <widget class="QWidget" name="Parameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="contrastSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="gainSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSlider" name="contrastSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="brightnessSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrast (0 to 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain (0 to 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure (-8 to 0)</string>
   </property>
  </widget>
  <widget class="QSlider" name="gainSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="exposureSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="minimum">
    <number>-8</number>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-8</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="brightnessSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="exposureSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>22</height>
    </size>
   </property>
   <property name="minimum">
    <number>-8</number>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Brightness (0 to 255)</string>
   </property>
  </widget>
  <widget class="QSlider" name="rotationSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotation (0 to 360)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rotationSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>22</height>
    </size>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
